l10n_br_account_payment_brcobranca/parser/cnab_file_parser.py

```diff
@@ -104,8 +104,8 @@ def _get_brcobranca_retorno(self, files):
     def _get_date_format(self, bank_name_brcobranca):
         # TODO: Idealmente o JSON de Retorno do BRCobranca deveria vir
         #  padronizado para não ser necessário ser feito esse tratamento aqui
-        if bank_name_brcobranca == "ailos":
-            # No Banco AILOS o formato da Data é completo com os 4 digitos.
+        if bank_name_brcobranca == "ailos" or bank_name_brcobranca == "santander":
+            # No Banco AILOS e Santander o formato da Data é completo com os 4 digitos.
```

Comment:
Não seria melhor assim?
```suggestion
        if bank_name_brcobranca in ["ailos","santander"]:
```

Reply:
Verdade, vou ajustar aqui, valeu @antoniospneto